src/pages/SykmeldingSide.tsx

```diff
@@ -26,17 +26,23 @@ const brodsmuler: Brodsmule[] = [
 ];
 
 const SykmeldingSide: React.FC = props => {
-    const { sykmelding, sykmeldingStatus } = useAppStore();
+    const { sykmelding, sykmeldingStatus, arbeidsgivere, sykmeldingUtenforVentetid } = useAppStore();
 
-    if (!sykmelding) {
+    if (!!!sykmelding || sykmeldingUtenforVentetid === null || arbeidsgivere === null) {
```

Comment:
!!! var en interessant variant. Det tilsvarer vel !

Reply:
https://stackoverflow.com/a/57828405/12159703
Fant dette for en stund siden. Trenger ikke bruke dersom det bare er forvirrende. Ingen funksjonell forskjell uansett